Formula/glibc.rb

```diff
@@ -152,12 +153,60 @@ def install
 
     # Delete bootstrap binaries after build is finished.
     rm_rf bootstrap_dir
+
+    # Add ld.so.conf (will be linked to HOMEBREW_PREFIX/etc/ld.so.conf)
+    (prefix/"etc/ld.so.conf").atomic_write <<~EOS
+      # This file is generated by Homebrew. Do not modify.
+      #{lib}  # ensure Homebrew Glibc always comes first
+      include #{etc}/ld.so.conf.d/*.conf
+    EOS
+    chmod "u=rw,go-wx", prefix/"etc/ld.so.conf"
```

Comment:
Given the number of times we do `chmod u=rw,go-wx`, it would be good to either extract this into a separate method, or at least extract the `u=rw,go-wx` into a variable.